src/polytope_blmos.jl

```diff
@@ -68,6 +131,41 @@ function bounded_compute_extreme_point(sblmo::ProbabilitySimplexSimpleBLMO, d, l
     return v
 end
 
+function bounded_compute_inface_extreme_point(sblmo::ProbabilitySimplexSimpleBLMO, direction, x, lb, ub, int_vars; kwargs...)
+    a = zeros(length(x))
+    if sblmo.N in lb
+        idx = findfirst(x->x==sblmo.N, lb)
+        a[idx] = sblmo.N
+        return a
+    end
+    min_val = Inf
+    min_idx = -1
+    for idx in eachindex(direction)
+        val = direction[idx]
+        if val < min_val && x[idx] > 0
+            min_val = val
+            min_idx = idx
+        end
+    end
+    a[min_idx] = 1.0
+    return a
+end
+
+function bounded_dicg_maximum_step(sblmo::ProbabilitySimplexSimpleBLMO, x, direction, lb, ub, int_vars; kwargs...)
+    return FrankWolfe.dicg_maximum_step(FrankWolfe.ProbabilitySimplexOracle{Float64}(), x, direction)
```

Comment:
You have to give the Probability Simplex the correct right side. Aka `N` in our case.